packages/lib/modules/vebal/vote/VoteList/VoteListFilters.tsx

```diff
@@ -124,6 +131,19 @@ export function VoteListFilters() {
                           toggledNetworks={toggledNetworks}
                         />
                       </Box>
+                      <Box as={motion.div} variants={staggeredFadeInUp}>
```

Comment:
don't we have all this as a component on pool list filters?

Reply:
No we don't, we could extract them if we refactor the shared filter state but I decided not do it yet.